src/tests/fixtures/projects/dummy-project.yaml

```diff
@@ -294,6 +294,16 @@ project-with-cloudfront-origins:
                     cookies:
                         - session_id
 
+project-with-fastly-minimal:
```

Comment:
we can test up to the template generation with this, which will be useful for all kind of customizations

Reply:
can we do any sort of dry run or validation like we can with Cloudformation (no matter how sucky it is) ?

Reply:
`terraform plan` does some sort of validation, but interacts with real infrastructure so it's a no-go for tests (and will eventually be executed as part of the process anyway).
https://www.terraform.io/docs/commands/validate.html does not do that much, but it's a start.